### PR TITLE
add back to top button

### LINKED ISSUE #1855: Back To Top Button
@Fraasi could you port your "Back To Top" Button from the new website to the old website? 

https://github.com/CodingTrain/project-trainsite-frontend/commit/d0a166076667bcc52d255f824f9fcb7f485d1d48#diff-cd7d7f55181a39702745095804ea7b6c

### PR DESCRIPTION
**Pull request by Fraasi**
Couldn't sleep, so here you go. I hope this approach is acceptable.
Haven't worked with this repo in a long time, it was kind of fun to navigate thru all the files 😸 
Closes #1855

### CODE REVIEW

**Comment by Fraasi** [comment]
```
  ● Web Editors › Metadata › VtYr6E4_M › has the right user

    FetchError: invalid json response body at https://editor.p5js.org/api/projects/VtYr6E4_M reason: Unexpected token < in JSON at position 0

      at node_modules/node-fetch/lib/index.js:272:32

  ● Web Editors › Metadata › VtYr6E4_M › has the right user

    TypeError: Cannot read property 'user' of undefined

      284 |         it('has the right user', () => {
      285 |           // Check that all web editor versions are on the right user
    > 286 |           expect(webEditorData.user.id).toBe('5b8578c76d67b5757e541f74');
          |                                ^
      287 |         })
      288 |       });
      289 |     }

      at Object.user (yaml.test.js:286:32)

  ● Web Editors › Metadata › zxruKuft9 › has the right user

    Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.Error: Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.

      at mapper (node_modules/jest-jasmine2/build/queueRunner.js:25:45)

  ● Web Editors › Metadata › zxruKuft9 › has the right user

    TypeError: Cannot read property 'user' of undefined

      284 |         it('has the right user', () => {
      285 |           // Check that all web editor versions are on the right user
    > 286 |           expect(webEditorData.user.id).toBe('5b8578c76d67b5757e541f74');
          |                                ^
      287 |         })
      288 |       });
      289 |     }

      at Object.user (yaml.test.js:286:32)

Test Suites: 1 failed, 1 total
Tests:       2 failed, 3480 passed, 3482 total
Snapshots:   0 total
Time:        31.542s
Ran all test suites.
npm ERR! Test failed.  See above for more details.
Exited with code 1
```

This has nothing to with my little button, what is this test?

**Comment by violetcraze** [comment]
I ran into a similar problem on [a test the other day](https://circleci.com/gh/CodingTrain/website/3685?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link). I think the web editor links are timing out. I went an manually checked the sketch to make sure it was under the coding train account. I wonder if we need to increase the timeout time or if there's something else that is wrong. In the mean time you could trigger a new build by changing a file committing and the fixing it and recommitting.

**Comment by Fraasi** [comment]
@violetcraze No luck, now I am getting different unrelated errors...
```
  *  284:1: ERROR: End tag : expected '>' (line 284)
- ./_site/Tutorials/9-additional-topics/index.html
  *  1053:1: ERROR: End tag : expected '>' (line 1053)
- ./_site/Tutorials/index.html
  *  4527:1: ERROR: End tag : expected '>' (line 4527)
- ./_site/index.html
  *  595:1: ERROR: End tag : expected '>' (line 595)
htmlproofer 3.13.0 | Error:  HTML-Proofer found 460 failures!
Exited with code 1
```

@gruselhaus Can you take a look at this, for me it looks like the tests are somehow broken?
Running this on localhost everything works fine.

File: _jekyll/includes/3-modules/back-to-top-button.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{% comment %}
+  A back to top button
+{% endcomment %}
+
+<div class="back-to-top-button" title="back to top">
+  <span>&nbsp;</span>
+  <span>&nbsp;</span
```
Comment (gruselhaus) [inline _jekyll/includes/3-modules/back-to-top-button.html]:
```suggestion
  <span>&nbsp;</span>
```

Comment (gruselhaus) [reply _jekyll/includes/3-modules/back-to-top-button.html]: This should make the trick.

**Comment by Fraasi** [comment]
Yeah, that did it. Thanks @gruselhaus

**Pull request merged**